materials/dplyr.md

```diff
@@ -110,3 +110,23 @@ query <- "SELECT year, month, day, genus, species
           ON surveys.species_id = species.species_id"
 tbl(portaldb, sql(query))
 ```
+
+* Queries and data manipulation functions return similar results with various headings.
+
```

Comment:
Can you walk me through you thinking on what you're hoping the students will get out of this new section?